lib/Transforms/FlattenMemRefs.cpp

```diff
@@ -336,10 +336,10 @@ struct FlattenMemRefCallsPass
 
     // Only run conversion on call ops within the body of the function. callee
     // functions are rewritten by rewriteFunctions=true. We do not use
-    // populateFuncOpTypeConversionPattern to rewrite the function signatures,
-    // since non-called functions should not have their types converted.
-    // It is up to users of this pass to define how these rewritten functions
-    // are to be implemented.
+    // populatefunc::FuncOpTypeConversionPattern to rewrite the function
```

Comment:
This looks accidental

Reply:
Yeah... format on save is overly aggressive